src/SavedSearch.php

```diff
@@ -68,6 +68,11 @@ public static function getTypeName($nb = 0)
         return _n('Saved search', 'Saved searches', $nb);
     }
 
+    public static function getSectorizedDetails(): array
+    {
+        return ['tools', \strtolower(__CLASS__)];
```

Comment:
It will be automatically compared with its lowercase version, no need to use `strtolower` here.

```suggestion
        return ['tools', self::class];
```